### PR TITLE
Update python binding

### LINKED ISSUE #1861: [Python] cannot import <module> from iota_wallet (unknown location)
## Bug description

Since the last update I cannot import iota_wallet like `from iota_wallet import StrongholdSecretManager, IotaWallet`
Error message `cannot import StrongholdSecretManager from iota_wallet (unknown location)`

image

The `__init.py__` in the latest iota-wallet library directory is missing
image

## Rust version

Which version of Rust are you running?

- Rust version: `rustc 1.67.1 (d5a82bbd2 2023-02-07)`

## Version

Which version of the library are you using?

- 1.0.0rc1
- 
image

## Hardware specification

What hardware are you using?

Language: en
Platform: linux
Platform Version: 5.15.89-1-MANJARO
Platform Architecture: x64
CPU Count: 24
Total Memory: 64231.6 MB
Free Memory: 38128.2 MB

## Steps To reproduce the bug

Explain how the maintainer can reproduce the bug.

I followed this installation procedure:

```
git clone https://github.com/iotaledger/wallet.rs && \
    cd wallet.rs/bindings/python && \
    pip install -r requirements-dev.txt && \
    pip install . && \
    rm -rf wallet.rs
```

then `from iota_wallet import StrongholdSecretManager, IotaWallet`

## Expected behaviour

iota_wallet modules to be imported correctly

## Actual behaviour

Impossible to import iota_wallet modules

## Errors

`__init.py__` is not copied to the `/lib/` directory of the `venv`

image

Once I copied the `__init.py__` file from the git repo to the `lib` folder the modules were imported correctly

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Update python binding

## Links to any relevant issues

Fixes #1861 

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Running `pip install .` and the added example on linux

Release workflow here https://github.com/Thoralf-M/wallet.rs/actions/runs/4210595495/jobs/7308386314

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/release-python-bindings.yml [hunk 1]
```diff
@@ -7,19 +7,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        # using macos-11 because of rocksdb
```
Comment (thibault-martinez) [inline .github/workflows/release-python-bindings.yml:10]: Is that still an issue with rocksdb 20?

Comment (Thoralf-M) [reply .github/workflows/release-python-bindings.yml:10]: Seems so https://github.com/iotaledger/wallet.rs/actions/runs/4192114912/jobs/7267286778

**Review by thibault-martinez — APPROVED** [review]

File: src/message_interface/account_method.rs [hunk 2]
```diff
@@ -285,6 +285,7 @@ pub enum AccountMethod {
     /// Expected response: [`Output`](crate::message_interface::Response::Output)
     PrepareOutput {
         options: OutputOptionsDto,
+        #[serde(rename = "transactionOptions")]
```
Comment (kwek20) [inline src/message_interface/account_method.rs:288]: we should stick with `rename_all = "camelCase"` when possible i think, is it here?

Comment (kwek20) [reply src/message_interface/account_method.rs:288]: never mind, the entire file is like this. smells like a cleanup job later!

**Review by kwek20 — APPROVED** [review]

File: src/message_interface/account_method.rs (hunk 2 above)
Comment (Thoralf-M) [reply src/message_interface/account_method.rs:288]: This doesn't work for nested enum fields, at least not like this
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=809f184f8234fec2c6d94a65a6d2ba7e

**Pull request merged**